### ISSUE TITLE
assertThat(...).isNotNull() does not qualify null-check for collections

### ISSUE THREAD

**Issue by jonatan-ivanov** [uber/NullAway#1219]
Similarly to https://github.com/uber/NullAway/issues/301 and https://github.com/uber/NullAway/issues/877,
AssertJ's `isNotNull` does not qualify as null-check for collections, here's a reproducer:

```java
@Test
void assertNotNull() {
    Collection<String> c = getCollection();
    assertThat(c).isNotNull();
    assertThat(c.size()).isEqualTo(0);
}

@Nullable Collection<String> getCollection() {
    return null;
}
```

which produces the following error for the last line of the `@Test` method with the latest version of NullAway (`0.12.7`):
```
java: [NullAway] dereferenced expression c is @Nullable
    (see http://t.uber.com/nullaway )
```

AssertJ has many `*Assert` classes and users can write their own too. Is it possible to scan every `*Assert` classes and interfaces in AssertJ and check if the `isNotNull` method belongs to one of them or to a child of one of them?

**Comment by msridhar** [comment]
@jonatan-ivanov can you try passing this flag to NullAway?

https://github.com/uber/NullAway/wiki/Configuration#assertions-in-test-libraries

Or are you already passing it?

**Comment by jonatan-ivanov** [comment]
https://github.com/uber/NullAway/issues/1224
